### ISSUE TITLE
inspec's "green" output displays as flashing red

### ISSUE THREAD

**Issue by hfinucane** [vito/elm-ansi#4]
As a more minimal test case, try `printf '\033[38;5;41m hello world\033[0m\n'` or this concourse task-
```
---
platform: linux

run:
  path: /usr/local/bin/bash
  args: ['-c', "printf '\\033[38;5;41m hello world\\033[0m\\n'"]

image_resource:
  type: docker-image
  source:
    repository: bash
```

in the Concourse discord, vito indicated it was probably 256-bit color related-
```
vito: @hfinucane you can open a bug on https://github.com/vito/elm-ansi - looks like it's missing 256-color support
```

Related tickets: https://github.com/inspec/inspec/issues/1700 https://github.com/jenkinsci/ansicolor-plugin/pull/96

**Comment by vito** [comment]
think this is fixed by #7

**Comment by aoldershaw** [comment]
The blinking part is fixed, but the actual colour that's used is still wrong since #7 didn't support full 8-bit colours - just the 0-15 range. See https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit
